TrainingCenterCRM.Api/Dto/PaginationDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TrainingCenterCRM.Api.Dto
+{
+    public class PaginationDto
```

Comment:
In my opinion, front end should send PageNumber and PageSize parameters. Offset and Limit are part of back end logic